contracts/WildcardSteward_v3.sol

```diff
@@ -141,7 +141,7 @@ contract WildcardSteward_v3 is Initializable {
         _;
     }
 
-    modifier collectPatronageAddress(address tokenPatron) {
+    modifier collectPatronagePatron(address tokenPatron) {
```

Comment:
This naming makes more sense to me, since its calling _collectPatronagePatron.

It also follows the precedence set above with collectPatronage and _collectPatronage